reference/programming_api/iresource.rst

```diff
@@ -63,7 +63,7 @@ IResource
         IEnumerable<IResource> ContainedResources { get; }
    }
 
-If you work with a POCO, you can use the extension method ToIResource(string informationModel) from Vonk.Core to adapt it to an IResource:
```

Comment:
If you use the `ToIResource()` method from `Vonk.Fhir.R4`, you don't need to supply the `informationModel` (that is inferred to be `Fhir4.0`). Likewise for `Vonk.Fhir.R3` and `R5`. See [here](https://github.com/FirelyTeam/Vonk/blob/54566967a23eeec00bbadc7ab89af4a99e92f82c/src/Vonk.Fhir.R4/FhirResourceExtensions.cs#L13).